Document for HTMLAnchorElement.text (#31887)
mdn/content: files/en-us/web/api/htmlanchorelement/text/index.md

---
title: "HTMLAnchorElement: text property"
short-title: text
slug: Web/API/HTMLAnchorElement/text
page-type: web-api-instance-property
browser-compat: api.HTMLAnchorElement.text
---

{{ApiRef("HTML DOM")}}

> The **`text`** property of the {{domxref("HTMLAnchorElement")}} reflects the {{HTMLElement("a")}} element text content.

Reviewer: Only attribute can be reflected.
```suggestion
The **`text`** property of the {{domxref("HTMLAnchorElement")}} represents the text inside the the element. 
```

> This property is the same as {{domxref("Node.textContent")}}.

Reviewer: ```suggestion
This property represents the same information as {{domxref("Node.textContent")}}.
```

> The **`text`** property of the {{domxref("HTMLAnchorElement")}} gets the {{HTMLElement("a")}} element text content.
> This property is same as {{domxref("Node.textContent")}}.

Reviewer 2: ```suggestion
The **`text`** property of the {{domxref("HTMLAnchorElement")}} reflects the {{HTMLElement("a")}} element text content.
This property is same as {{domxref("Node.textContent")}}.
```

the property is not read-only

> This property is same as {{domxref("Node.textContent")}}.

Reviewer 3: ```suggestion
This property is the same as {{domxref("Node.textContent")}}.
```

## Value

A string.

## Example

```html
<a id="exampleLink" href="https://example.com">Example Link</a>
<p class="text"></p>
```

```css
#exampleLink {
  font-size: 1.5rem;
}
```

```js
const anchorElement = document.getElementById("exampleLink");
const pTag = document.querySelector(".text");

> pTag.textContent = anchorElement.text + " " + "text property";

Reviewer: Let's makes it clearer. Also template string are much more readable in this case:
```suggestion
pTag.textContent =  `Text property: ${anchorElement.text}`;
```

```

> 
> ## Result
> 
> {{EmbedLiveSample("Example",100,100)}}

Reviewer 2: ```suggestion

{{EmbedLiveSample("Example",100,100)}}
```

*Result* section is not necessary

Author: I kept it cus reader might check Node.textContent

Reviewer 2: Or *###Result*

Author: My bad I didn't notice
